cpp/include/cudf_test/column_utilities.hpp

```diff
@@ -199,8 +199,14 @@ template <>
 inline std::pair<thrust::host_vector<std::string>, std::vector<bitmask_type>> to_host(column_view c)
 {
   auto strings_data = cudf::strings::create_offsets(strings_column_view(c));
-  thrust::host_vector<char> h_chars(strings_data.first);
-  thrust::host_vector<size_type> h_offsets(strings_data.second);
+  thrust::host_vector<char> h_chars(strings_data.first.size());
+  thrust::host_vector<size_type> h_offsets(strings_data.second.size());
+  CUDA_TRY(
+    cudaMemcpy(h_chars.data(), strings_data.first.data(), h_chars.size(), cudaMemcpyDeviceToHost));
+  CUDA_TRY(cudaMemcpy(h_offsets.data(),
```

Comment:
I'm curious, why do you change to do manual copies here?

Reply:
A `thrust::host_vector` constructor does not accept an `rmm::device_uvector` type.

Reply:
Can we use `thrust::host_vector<size_type> h_offsets(strings_data.second.begin(), strings_data.second.end());`?

Reply:
No. I tried this initially and it segfaults.